tests/Unit/RmDirStepTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use WordPress\Blueprints\Model\Builder\RmDirStepBuilder;
+use WordPress\Blueprints\Runner\Step\RmDirStepRunner;
+
+it('should remove a directory', function () {
+    $path = __DIR__ . "/dir";
+    mkdir($path);
+    expect( file_exists($path)) -> toBeTrue();
+
+    $stepBuilder = new RmDirStepBuilder();
```

Comment:
Let's ditch testing the builders for now while they still settle and go directly with the model objects, e.g. `RmDirStep` in this case. So this example would be:

```suggestion
    $step = new RmDirStep();
    $step->path = $path;
```

Reply:
Done.